docs/platform/howto/project-support-center.rst

```diff
@@ -29,10 +72,63 @@ To create support tickets in the `Aiven Console <https://console.aiven.io/>`_:
     * Any error messages
     * Any languages or frameworks you are using
 
-#. Select the affected project, service, and a severity level.
+#. Optional: Upload files such as screenshots, logs, or :ref:`HAR files <create-har-files>`.
+   
+   .. important::
+        Aiven support will never ask you to provide sensitive data such as passwords or personal information. Remove or replace sensitive data in files that you attach to a support ticket.
 
 #. Click **Create ticket**. 
 
-The new support ticket is sent to our support team for review and you also receive a confirmation email. `Response times <https://aiven.io/support-services>`_ vary by case severity and support tier. 
+You can track the status of your tickets on the **My tickets** page. `Response times <https://aiven.io/support-services>`_ vary by case severity and support tier. If you are not satisfied with the processing of your ticket, add ``#escalate`` in the comments.
+
+
+Add participants to a support ticket
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to give every organization user access to all support tickets in your organization contact your account team.
+
+To add Aiven users to a support ticket:
+
+#. On the **My tickets** page, open the ticket.
+
+#. Click **Add to conversation**.
+
+#. Add the email addresses in the **CC** field separated by a space. This must be the same email address they use to log in.
+
+#. Enter a comment and click **Submit**.
+
+
+Get notifications for all support tickets 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Super admin can get notifications for updates on all tickets in their organization. 
+
+#. Click **My tickets**.
+
+#. On the **Tickets in my organization** tab, click **Follow all tickets**.
+
+You will get email notifications for all updates on both existing and new tickets. You can unfollow them at any time.
+
+
+.. _create-har-files:
+
+Create HAR files
+~~~~~~~~~~~~~~~~~
+
+Aiven support may need information about the network requests that are generated in your browser when you experience a problem. Browsers can capture a log of these network requests in a HAR (HTTP Archive) file. 
```

Comment:
Perhaps to avoid confusion:

```suggestion
Aiven Support may need information about the network requests that are generated in your browser when you experience a problem. Browsers can capture a log of these network requests in a HAR (HTTP Archive) file. 
```